endless/eosprofile-private.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2017  Endless */
```

Comment:
Should probably be a full copyright/license header.

Reply:
I've followed what's already there, but I can add this